jetson/percep/perception.hpp

```diff
@@ -118,7 +118,6 @@ class obstacle_return {
     }
     
     bearing = in.bearing;
-    distance = in.distance;
     return *this;
```

Comment:
Why not just return `this`? Seems more light weight? Could also be `const` if you don't want `this` modified.